tests/login.spec.ts

```diff
@@ -0,0 +1,31 @@
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('/auth/login');
+});
+//add tests for login page
+test('Check that site has a title', async ({ page }) => {
+    await expect(page).toHaveTitle('Login - Practice Software Testing - Toolshop - v5.0');
+  });
+
+test('Fill in the fields on the login page', async ({ page }) => {
+
+    const emailTest = 'customer@practicesoftwaretesting.com'
+    const emailField = page.getByTestId('email')
+    await expect(emailField).toBeVisible();
```

Comment:
Actually, we do not need this assertion, since playwright wait automatically until element is visible. PLease check docs - https://playwright.dev/docs/actionability 
![Uploading image.png…]()